app/src/main/java/org/oppia/android/app/topic/revisioncard/RevisionCardFragmentPresenter.kt

```diff
@@ -74,4 +88,10 @@ class RevisionCardFragmentPresenter @Inject constructor(
       oppiaLogger.createRevisionCardContext(topicId, subTopicId)
     )
   }
+
+  override fun onConceptCardLinkClicked(view: View, skillId: String) {
+    ConceptCardFragment
+      .newInstance(skillId)
+      .showNow(fragment.childFragmentManager, CONCEPT_CARD_DIALOG_FRAGMENT_TAG)
```

Comment:
The const `CONCEPT_CARD_DIALOG_FRAGMENT_TAG` is defined in `StatePlayerRecyclerViewAssembler`.
Considering that now this fragment is getting used outside of `player` too. I think we should shift this to `ConceptCardFragment`

Reply:
Nice catch, and good suggestion. Moved to the fragment's companion object & tested affected tests locally to ensure nothing broke.

Reply:
Sounds good.